### PR TITLE
Adding Spanish language README.

### PR DESCRIPTION
**Pull request by tonyskapunk**
Includes the link in main README to the Spanish README.

---
Fix: #1417

### CODE REVIEW

**Comment by SirCmpwn** [comment]
@jdiez17 can you take a look at this

**Comment by SirCmpwn** [comment]
Please review the guidelines, @tonyskapunk https://github.com/swaywm/sway/issues/1318

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+* libinput >= 1.6.0
+* libcap
+* asciidoc
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 *
+* pam **
+* imagemagick (requerido para captura de imágenes con swaybrab)
```
Comment (angvp) [inline README.es.md]: typo here it's swaygrab

File: README.es.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    sudo setcap "cap_sys_ptrace,cap_sys_tty_config=eip" /usr/bin/sway
+
+En sistemas sin logind, se necesita habilitar el "suid" en el binario de sway:
+
+    sudo chmod a+s /usr/local/bin/sway
+
+## Configuración
+
+Si ya haces uso de i3 sólo copia tu configuracirión de i3 a
```
Comment (angvp) [inline README.es.md]: configuración another typo

**Comment by tonyskapunk** [comment]
Thanks for looking, I've fixed the typos in the last commit, I can rebase at any time since this branch is out of date, before doing that I want to ensure the commits look right.

**Comment by Laaas** [comment]
....what happened to this?

File: README.es.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+# sway [![](https://api.travis-ci.org/swaywm/sway.svg)](https://travis-ci.org/swaywm/sway) [![Donate with fosspay](https://drewdevault.com/donate/static/donate-with-fosspay.png)](https://drewdevault.com/donate?project=4)
+
+[**English**](https://github.com/swaywm/sway/blob/master/README.md#sway--) - [日本語](https://github.com/swaywm/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/swaywm/sway/blob/master/README.de.md#sway--) - [Ελληνικά](https://github.com/swaywm/sway/blob/master/README.el.md#sway--) - [Español](https://github.com/swaywm/sway/blob/master/README.es.md#sway--) - [Français](https://github.com/swaywm/sway/blob/master/README.fr.md#sway--) - [Українська](https://github.com/swaywm/sway/blob/master/README.uk.md#sway--) - [Italiano](https://github.com/swaywm/sway/blob/master/README.it.md#sway--) - [Português](https://github.com/swaywm/sway/blob/master/README.pt.md#sway--) -
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--)
+
+"**S**irCmpwn's **Way**land compositor" es un gestor de composición de ventanas
+compatible con i3 [Wayland](http://wayland.freedesktop.org/) **en processo de
```
Comment (miguelbernadi) [inline README.es.md]: Should be `gestor de composción de ventanas Wayland compatible con i3` as i3 itself is unrelated to Wayland.

File: README.es.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+desarrollo**.
+Lee el ["FAQ"](https://github.com/swaywm/sway/wiki). Únete al canal de
+["IRC"](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway en
+irc.freenode.net).
+
+**Nota**: Sway se encuentra actualmente "congelado"("frozen"), es decir que
```
Comment (miguelbernadi) [inline README.es.md]: `"congelado"("frozen")` seems missing a space between the `"` and `(`. I'd expect to have one there.

File: README.es.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+[![](https://sr.ht/ICd5.png)](https://sr.ht/ICd5.png)
+
+
+Para el soporte de desarrollo de Sway se puede contribuir a [SirCmpwn's
+Patreon](https://patreon.com/sircmpwn) o se puede contribuir mediante el
+formato de recompensas ["bounties"](https://github.com/swaywm/sway/issues/986)
+para características especificas.
+Cualquier persona es bienvenida a contribuir y reclamar una recompensa.
+Adicionalmente se pueden crear recompensas("bounties") para cualquier
```
Comment (miguelbernadi) [inline README.es.md]: `recompensas("bounties")` would also expect a whitespace before the `(`.

File: README.es.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+característica que se desee desarrollar. En general el uso de Patreon es más
+útil para el sustento de Sway.
+
+## Ayuda en Español
+
+Al momento no hay ayuda en Español en IRC, [tonyskapunk](https://github.com/tonyskapunk)
+puede proveer ayuda en Español en GitHub en el uso horario [UTC-6](https://en.wikipedia.org/wiki/UTC%E2%88%9206:00)
```
Comment (miguelbernadi) [inline README.es.md]: `huso horario`.

File: README.es.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+- [IPC - soporte de características](https://github.com/swaywm/sway/issues/98)
+- [i3bar - soporte de características](https://github.com/swaywm/sway/issues/343)
+- [i3-gaps - soporte de características](https://github.com/swaywm/sway/issues/307)
+- [características de seguridad](https://github.com/swaywm/sway/issues/984)
+
+## Instalación
+
+### Por medio de paquetes
+
+Sway esta disponible en muchas distribuciones. Intenta instalar el paquete
```
Comment (miguelbernadi) [inline README.es.md]: `está`

File: README.es.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+"sway" en tu distribución. Si no esta disponible revisa
```
Comment (miguelbernadi) [inline README.es.md]: `está`

File: README.es.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+[ésta wiki](https://github.com/swaywm/sway/wiki/Unsupported-packages)
+para información de la instalación en tu distribución.
+
+Si estas interesado en crear un paquete de Sway para tu distribución entra
```
Comment (miguelbernadi) [inline README.es.md]: `estás`

File: README.es.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+### Por medio de paquetes
+
+Sway esta disponible en muchas distribuciones. Intenta instalar el paquete
+"sway" en tu distribución. Si no esta disponible revisa
+[ésta wiki](https://github.com/swaywm/sway/wiki/Unsupported-packages)
+para información de la instalación en tu distribución.
+
+Si estas interesado en crear un paquete de Sway para tu distribución entra 
+al canal de IRC o envía un correo(en Inglés) a sir@cmpwn.com para más
```
Comment (miguelbernadi) [inline README.es.md]: `correo(en Inglés)` missing whitespace before `(`.

File: README.es.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 *
+* pam **
+* imagemagick (requerido para captura de imágenes con swaygrab)
+* ffmpeg (requerido para captura de video con swaygrab)
+
+_\*Sólo requerido para swaybar, swaybg, and swaylock_
```
Comment (miguelbernadi) [inline README.es.md]: `and swaylock` should be `y swaylock`.

File: README.es.md (hunk 3 above)
Comment (tonyskapunk) [reply README.es.md]: fixed

File: README.es.md (hunk 4 above)
Comment (tonyskapunk) [reply README.es.md]: fixed

File: README.es.md (hunk 5 above)
Comment (tonyskapunk) [reply README.es.md]: It is a  `[` and not a `(`, is a markdown link.

File: README.es.md (hunk 6 above)
Comment (tonyskapunk) [reply README.es.md]: fixed

File: README.es.md (hunk 7 above)
Comment (tonyskapunk) [reply README.es.md]: fixed

File: README.es.md (hunk 8 above)
Comment (tonyskapunk) [reply README.es.md]: fixed

File: README.es.md (hunk 9 above)
Comment (tonyskapunk) [reply README.es.md]: fixed

File: README.es.md (hunk 10 above)
Comment (tonyskapunk) [reply README.es.md]: fixed

File: README.es.md (hunk 11 above)
Comment (tonyskapunk) [reply README.es.md]: also fixed.

**Comment by tonyskapunk** [comment]
Thanks for the review @miguelbernadi I've fixed all the typos in my last commit.

File: README.es.md (hunk 5 above)
Comment (miguelbernadi) [reply README.es.md]: In line 29 it's not a markdown link. `Adicionalmente se pueden crear recompensas("bounties") para cualquier`

**Comment by miguelbernadi** [comment]
Thanks for the effort @tonyskapunk ! You'll still need to rebase over master before merging?

File: README.es.md (hunk 5 above)
Comment (tonyskapunk) [reply README.es.md]: Ah I see the  problem, this PR was created when the content of the README was a bit diff, this commit removed the reference to the bounties f0ddf6d74

**Comment by tonyskapunk** [comment]
Yes I'll rebase once everything is good, that way is easier for the reviewer(s).

Regarding the "bounties" comment, I see the README has changed considerably, I think it would be better to match it to the actual one, instead of pushing with the old content.  Thoughts?

**Comment by tonyskapunk** [comment]
I see that the README on each language have been changed a lot since this PR, I can help with the .es version of them if those are reflecting the version 1.0.

**Comment by ddevault** [comment]
I would appreciate that.

**Pull request closed without merging**